lighthouse/datadog_checks/lighthouse/lighthouse.py

```diff
@@ -11,73 +11,120 @@
 
 class LighthouseCheck(AgentCheck):
     def check(self, instance):
-        lighthouse_url = instance.get('url')
+        backward_compatible_lighthouse_url = instance.get('url')
+        lighthouse_urls = instance.get('urls', [])
         lighthouse_name = instance.get('name')
         extra_chrome_flags = instance.get('extra_chrome_flags', [])
+        form_factor = instance.get('form_factor')
```

Comment:
Can we alert the user  if `form_factor` is not `mobile` or `desktop`, and set `form_factor` to `None` if so?